inst/extdata/packages/ENMTML.yaml

```
@@ -0,0 +1,42 @@
+name : "ENMTML"
+title: "Create and process Ecological Niche, including several pre- and post-processing methods"
+version: 1.0.0
+author: "André F.A. Andrade, Santiago J.E. Velazco, Paulo De Marco"
+maintainer: "André F.A. Andrade <andrefaandrade@gmail.com>"
+cran: no
+github: https://andrefaa.github.io/ENMTML
+description: "ENMTML is a package to create and
+      process Ecological Niche Models. The main
+      purpose of this package is to assemble all this
+      methodological complexity spread over several
+      papers and bring it into the spotlight in a
+      simple way for people not used to the details
+      of ENMs. The package contains several
+      alternatives to different methodological steps,
+      e.g., pseudo-absence allocation and accessible
+      area delimitation, formulated within a single
+      function, to make it accessible for people not
+      used to the programming environment."
+occ_acquisition: no
+occ_cleaning: yes
+data_integration: no
+env_collinearity: yes
+env_process: no
+bias: yes
```

```suggestion
bias: no
```
Argument `thin_occ` is within function `ENMTML()`. I'd characterize thinning as "cleaning" more than assessing bias. @sylvainschmitt, what do you think?

---

Yes I was hesitating on this one too. And I think you are right. In my case for `SSDM` we rely on `spThin` for spatial thinning but I did not include it in the bias field: https://github.com/sylvainschmitt/sdmverse/blob/cf4e0f8e23db8b1ee024d46cf240daa196a371f2/inst/extdata/packages/SSDM.yaml#L22
Indeed thinning is more cleaning occurrences than assessing bias as you pointed out. Secondly, the aim of sdmverse is to illustrate the complementarity and differences among packages. So I think we should focus on core functionalities, and avoid answer yes for fields when it is only minor functionalities (because if everybody have yes everywhere sdmverse becomes useless).

Anyway this is great that you pointed it out, and it illustrate how it is important that we always try to have at least two reviewers.

---

I'm liking this system. Thanks for setting it up.